packages/syftcli/setup.py

```diff
@@ -22,17 +23,16 @@
     long_description_content_type="text/plain",
     version=__version__,
     author="OpenMined <info@openmined.org>",
-    packages=find_packages(where="src"),
-    package_dir={"": "src"},
+    packages=find_packages(),
     install_requires=packages,
     extras_require={"dev": dev_packages, "build": build_packages},
     include_package_data=True,
     entry_points={
         "console_scripts": [
-            "syft = syftcli.cli:app",
-            "syft-cli = syftcli.cli:app",
-            "syftcli = syftcli.cli:app",
-            "syftctl = syftcli.cli:app",
+            "syft = src.cli:app",
```

Comment:
MAYBE: I gave it a lot of thought, Shall we go with syftcli instead of src?, for the inner package.
Importing src as the main package feels a lot unintuitive and having it same as package name , would be easy for new people exploring the codebase and imports statements, would point directly to the package name of the tool

like

```
packages
├── grid
├── syft
└── syftcli
    ├── syftcli
    │   ├── common/            # shared ccode examples
    │   │   ├── utils.py       
    │   │   ├── tools.ps
    │   │   └── popen.py
    │   ├── deploy/            # Nested command Example: syftcli deploy (docker|local|azure)
    │   │   ├── __init__.py
    │   │   ├── docker.py
    │   │   ├── local.py
    │   │   └── azure.py
    │   ├── hello.py           # Simple command. Example: syftcli hello
    │   ├── version.py         # Simple command. Example syftcli version
    │   └── cli.py             # CLI Entry Point
    ├── setup.cfg
    ├── Dockerfile
    └── README.md
```

Reply:
Updated from "src" to "syftcli"